src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -453,14 +470,31 @@ def import_industrial_gas_demand_time_series(egon_etrago_load_gas):
     )
 
 
-def insert_industrial_gas_demand():
-    """Overall function for inserting the industrial gas demand
+def download_industrial_gas_demand():
+    """Download the industrial gas demand data from opendata.ffe database."""
+    correspondance_url = (
+        "http://opendata.ffe.de:3000/region?id_region_type=eq.38"
+    )
 
-    Returns
-    -------
-    None.
-    """
+    # Read and save data
+    result_corr = requests.get(correspondance_url)
+    pd.read_json(result_corr.content).to_json("region_corr_ind_demand.json")

```

Comment:
This file could be stored in datasets/gas_data

Reply:
Sidenote: it might make sense to store temp files in the working dir for housekeeping purposes? :)
([example 1](https://github.com/openego/eGon-data/blob/6e0da9c2885980ad33ddb07a16fa83d06034257a/src/egon/data/datasets/emobility/motorized_individual_travel/__init__.py#L101-L107), [example 2](https://github.com/openego/eGon-data/blob/6e0da9c2885980ad33ddb07a16fa83d06034257a/src/egon/data/datasets/emobility/motorized_individual_travel/__init__.py#L254-L255), `WORKING_DIR` is a subdir of the actual working dir, defined [here](https://github.com/openego/eGon-data/blob/6e0da9c2885980ad33ddb07a16fa83d06034257a/src/egon/data/datasets/emobility/motorized_individual_travel/helpers.py#L11))

Reply:
Thanks to both of you. In this sense it does actually make sense, to keep those files permanently, so they do not need to be redownloaded because the raw data are directly read and processed by one single task.

Reply:
Fixed by https://github.com/openego/eGon-data/pull/639/commits/71e94fad9dcaf7124ffdaf8054b313f743fe984d